app/models/blacklight_config.rb

```diff
@@ -24,7 +24,8 @@ def configurations
       DefaultConfiguration.new(configuration),
       SearchFieldConfiguration.new(configuration),
       ShowActions.new(configuration),
-      ThumbnailConfiguration.new(configuration)
+      ThumbnailConfiguration.new(configuration),
+      Gallery.new(configuration)
```

Comment:
Just a question. Do we want to keep the names of these consistent?  _____Configuration. Or does it really matter?